### DISCUSSION TITLE
call by reference syntax

### DISCUSSION THREAD

**Discussion by rvjansen** [adesutherland/CREXX#240] [General]
Interestingly, I found a reference to 'call by reference' syntax in a 1993 Mike Cowlishaw 'Future Rexx' presentation.
![Screenshot 2021-12-17 at 16 55 31](https://user-images.githubusercontent.com/12018881/146607138-068e5405-5d0a-4c86-b601-c1577271eda4.png)

**Comment by adesutherland** [comment]
I like alias

**Comment by rvjansen** [comment]
me too, but it seems that 'use arg' has a foothold in ooRexx
